Gir_GstPbutils.toml

```diff
@@ -141,3 +141,93 @@ trait = false
 name = "GstPbutils.DiscovererContainerInfo"
 status = "generate"
 trait = false
+
+[[object]]
+name = "GstPbutils.EncodingProfile"
+status = "generate"
+    [[object.function]]
+    name = "is_equal"
+    ignore = true
+
+    # Ignore all setters, making it imutable. A builder will be provided.
+    [[object.function]]
+    name = "new"
+    ignore = true
+    [[object.function]]
+    pattern = "set_.*"
+    ignore = true
+
+    [[object.function]]
+    name = "copy"
+        [object.function.return]
+        nullable = false
+
+    [[object.function]]
+    name = "get_input_caps"
+        [object.function.return]
+        nullable = false
+
+    [[object.function]]
+    name = "get_format"
+        [object.function.return]
+        nullable = false
+
+[[object]]
+name = "GstPbutils.EncodingContainerProfile"
+status = "generate"
+
+    # Make it imutable, only able to be constructed for a builder
+    [[object.function]]
+    name = "new"
+    ignore = true
+
+    [[object.function]]
+    name = "add_profile"
+    ignore = true
+
+[[object]]
+name = "GstPbutils.EncodingTarget"
+status = "generate"
+
+    [[object.function]]
+    name = "add_profile"
+        [object.function.return]
+        bool_return_is_error = "Failed to add profile"
```

Comment:
Should this maybe also be a builder or is it acceptable to add profiles to the target at any time?